src/sidebar/components/test/tag-editor-test.js

```diff
@@ -295,8 +287,9 @@ describe('TagEditor', function () {
       assertAddTagsFail();
     });
 
-    it('should not a tag when pressing "Tab" and input typed is not a suggestion', () => {
+    it('should not a tag when pressing "Tab" and there are no suggestions', () => {
```

Comment:
Nit: Missing a word in this test description (maybe it should say "should not ADD a tag...", emphasis aside...?)

Reply:
thanks

Reply:
thanks!